ci-operator/config/openshift-eng/agent-qe-infra/tools/generate_cron_bm_abi.py

```
@@ -0,0 +1,197 @@
+
+    if not job_weight_map:
+        print("No matching jobs discovered based on constraints. Exiting.")
+        return
+
+    # Loop back through files to optimize matching cron expressions
+    for dir_path in valid_paths:
+        for yaml_file in dir_path.rglob("*.yaml"):
+            try:
+                lines = yaml_file.read_text().splitlines()
+            except Exception:
+                continue
```

Ruff (0.15.17)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)